Add landcover layer

planemad opened the pull request:
## Summary

- Add 'Landcover' layer from [Bhuvan](https://github.com/ramSeraph/indian_land_features/releases/tag/landuse)


<img width="747" alt="Screenshot 2025-03-27 at 8 13 43 AM" src="https://github.com/user-attachments/assets/7d0a763e-6e65-41d3-8d92-a135ffdb68a1" />


## Testing Strategy

Live preview: https://amche.in/dev/?layers=landcover,mask

cc @ramseraph

marlonrodrigues reviewed (commented):
Should we explain what Description 1 and Description 2 mean in the legend?

Row of map icons appear vertically stretched in this version 

![Screenshot 2025-03-27 at 12 27 12 PM](https://github.com/user-attachments/assets/e2088a5d-0602-4177-b604-830f690643b2)

ramSeraph commented:
Ok, I found a better version landcover dataset with more consistent fields and better coverage.. I will be putting it up shortly( I think the download is going to take a week), I would need some feedback on what detail to keep at what zoom level while tiling it.  Your call on whether to keep the current version on.

planemad commented:
@marlonrodrigues fixed the issues:
- Fixed squished icons in popup
- Updated field descriptions for landcover
- Add vector labelling support for layers
- Replace layer checkboxes with switches
- Add several new layers ✨ 

<img width="1062" alt="Screenshot 2025-04-03 at 12 30 10 AM" src="https://github.com/user-attachments/assets/3966292b-d6fb-4b0e-b5fa-ecb0456b0d6b" />
